src/functions/getSubheaderList.js

```diff
@@ -1,17 +1,11 @@
 import React from "react";
 import { List } from "../components/List.js";
-
-export function getSubheaderList(dimensions, dimensionsCount, italicsInput) {
-  const listItems = [];
-  for (let listItem = 0; listItem < dimensionsCount; listItem++) {
-    listItems.push(
-      <List
-        message="Select zero to N "
-        columns={dimensions[listItem]}
-        styling={italicsInput}
-        key={listItem}
-      />
-    );
-  }
-  return listItems;
-}
+export const getSubheaderList = (dimensions, italicsInput) =>
```

Comment:
this should also be `<Component />`

Reply:
functions should **never** have a react dependency

note: `functions != functionalComponents`

Reply:
f5633b5b82caa46b2d66be4bb19a999b9d7db1c2